apple/Sources/FerrostarMapLibreUI/Extensions/EdgeInsets.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUICore
+
+extension EdgeInsets {
+    /// Given a view's insets, returns the insets for the child view, such that
+    /// the child's insets, accumulated with the parents insets, will be at least `minimumInset`
+    func collapsedChildInsets(minimumInset: CGFloat) -> EdgeInsets {
```

Comment:
I tried really hard to think of a better name and this is the best I came up with. Suggestions welcome!

Reply:
I thought harder and reworked https://github.com/stadiamaps/ferrostar/pull/258/commits/961343f03bbdfc62cf7879f2fc85e67d6f460503 to be more self contained.